exercises/dnd-character/DndCharacter.fs

```diff
@@ -8,12 +8,7 @@ let modifier x =
 let ability() = 
     failwith "You need to implement this function."
 
-type DndCharacter() =
-    member __.Strength with get() = failwith "You need to implement this function."
-    member __.Dexterity with get() = failwith "You need to implement this function."
-    member __.Constitution with get() = failwith "You need to implement this function."
-    member __.Intelligence with get() = failwith "You need to implement this function."
-    member __.Wisdom with get() = failwith "You need to implement this function."
-    member __.Charisma with get() = failwith "You need to implement this function."
-    member __.Hitpoints with get() = failwith "You need to implement this function."
+// TODO: define the Character type
 
+let mkCharacter() =
```

Comment:
Maybe `createCharacter ()`? There's no reason to abbreviate things here.

Reply:
You're right, I'll update accordingly